### DISCUSSION TITLE
Google Maps not showing in the emulator

### DISCUSSION THREAD

**Discussion by moving-bits** [cgeo/cgeo#13607] [Q&A]
I have a persisting problem with Google Maps in the emulator: Only a blank page is shown when opening GM:

![image](https://user-images.githubusercontent.com/3754370/200117609-209591eb-55f5-4167-ba1e-ef9b551df53f.png)

I've cross-checked the API key, the app's fingerprint and API restrictions - looks fine.
It has stopped working from one day to the next.

This happens in the emulator only, though. When I deploy the same debug build to a physical device, GM shows up.

Does anyone have an idea on how to resolve this?
I would really like to be able to test GM in the emulator again, as my physical testing device is so sloooooow...

**Comment by Lineflyer** [comment]
Are you sure you are using an emulator image with Google services?
AFAIR there are images available each with and without Google services for each API level.
